source/extensions/filters/listener/proxy_protocol/BUILD

```diff
@@ -31,9 +32,10 @@ envoy_cc_library(
     ],
 )
 
-envoy_cc_library(
+envoy_cc_extension(
     name = "config",
     srcs = ["config.cc"],
+    security_posture = "unknown",
```

Comment:
untrusted downstream for sure. We should have a fuzzer for this one also. cc @asraa

Reply:
I've added this to the fuzzer wishlist backlog that @asraa is tracking.